startReactCommerce.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+git checkout main
```

Comment:
Seems odd to always force the user to the main branch.

Reply:
We `git checkout -` to return the user to their old branch afterwards. The point is to make sure their main branch is up to date in case they merge or want to checkout a non-local branch.

Reply:
TIL about `git checkout -`.